rosdep/base.yaml

```
@@ -8066,6 +8066,11 @@ xtensor:
   ubuntu:
     bionic: [xtensor-dev]
     jammy: [libxtensor-dev]
+xsimd:
+  debian: [libxsimd-dev]
+  fedora: [xsimd-devel]
+  ubuntu:
+    jammy: [libxsimd-dev]
```

I think the following syntax is more future-proof:

```suggestion
    "*": [libxsimd-dev]
    focal: null
```

---

Sorry, looks like we also need to add a null entry for bionic as well.